docs/components/integration-framework/connectors/custom-built-connectors/connector-sdk.md

```diff
@@ -4,17 +4,677 @@ title: Connector SDK
 description: Introduction to the Connector SDK.
 ---
 
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
 :::note
 The **connector SDK** is in developer preview and subject to breaking changes. Use at your own risk.
 :::
 
-The **connector SDK** allows you to develop your own **connectors**. With the help of the SDK you can perform common operations, such as:
+The **connector SDK** allows you to [develop custom connectors](#creating-a-custom-connector)
+using Java code. You can focus on the logic of the connector, conveniently test it locally, and
+reuse its runtime logic in multiple environments. The SDK achieves this by abstracting from
+Camunda Platform 8 internals that usually come with
+[job workers](../../../concepts/job-workers.md).
+
+The SDK provides API for common connector operations, such as:
+
+- Fetching and deserializing input data
+- Validating input data
+- Replacing secrets in input data
+
+Beyond that, the SDK allows for [convenient testing](#testing) of your connector behavior and
+[executing it in the environments](#runtime-environments) that suit your use cases best.
+
+## Creating a custom connector
+
+Using the connector SDK, you can create environment-agnostic, reusable connector runtime behavior.
+This section will outline how to set up a connector project, test it, and run it locally.
+
+### Setup
+
+We recommend starting developing a connector using our
```

Comment:
Minor tweak: 
"We recommend to start developing a connector using our"

Reply:
Funny enough, Grammarly corrected me on this the other way 'round 😄

Reply:
Gotcha. I updated the text.

Reply:
Updated